setup.py

```diff
@@ -28,10 +28,9 @@
         "flaky>=3.7.0",
         "hypothesis>=3.31.2",
         "importlib-metadata<5.0;python_version<'3.8'",
-        "pytest>=6.2.5",
+        "pytest==7.2.2",
```

Comment:
If possible, I'd rather bump `pytest>=7.0.0` to allow a wider range. Does that work here?